js/background.js

```diff
@@ -324,6 +324,15 @@
         storage.put('message-ttl', ttl);
       },
 
+
+      // getZoomFactor: () => storage.get('zoom-factor-setting', 48),
```

Comment:
Can we get these commented out lines removed if we're not going to be using it.

Reply:
all removed